metadata.yaml

```diff
@@ -24,6 +24,11 @@ issues: https://github.com/canonical/self-signed-certificates-operator/issues
 provides:
   certificates:
     interface: tls-certificates
+  send-ca-cert:
+    interface: mutual_tls
```

Comment:
Would it not be better to create a specific interface for sharing certificates to trust? I think this will get confusing, and if we want to refactor this down the line it will break other charms.